updater/fetchers/archlinux.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2015, 2016 clair authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fetchers
+
+import (
+	"bufio"
+	"io"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/coreos/clair/database"
+	"github.com/coreos/clair/updater"
+	cerrors "github.com/coreos/clair/utils/errors"
+)
+
+const (
+	archLinuxCVEURL      = "https://wiki.archlinux.org/api.php?action=query&titles=CVE&format=txt&prop=revisions&rvlimit=1&rvprop=content"
+	archlinuxUpdaterFlag = "archlinuxUpdater"
+	tokensRegexp         = "{|}|CVF|PKG|Pkg|pkg|\\[|\\]"
+)
+
+type SecurityAdvisory struct {
+	Name string
+	URL  string
+}
+
+// ArchCVE represents a CVE for Arch Linux
+type ArchCVE struct {
```

Comment:
Let's choose one way to say Arch Linux and be consistent. Either just `Arch` or `ArchLinux` is what I'd recommend.